SPECS-CROSS/binutils_cross/binutils_cross.spec

```diff
@@ -0,0 +1,248 @@
+
+%files -f %{name}.lang
+%defattr(-,root,root)
+%license COPYING
+#%%{_cross_prefix}%%{_cross_bindir}/dwp
+#%%{_cross_prefix}%%{_cross_bindir}/gprof
+%{_cross_prefix}%{_cross_bindir}/ld.bfd
+#%%{_cross_prefix}%%{_cross_bindir}/ld.gold
+#%%{_cross_prefix}%%{_cross_bindir}/c++filt
+%{_cross_prefix}%{_cross_bindir}/objdump
+%{_cross_prefix}%{_cross_bindir}/as
+%{_cross_prefix}%{_cross_bindir}/ar
+%{_cross_prefix}%{_cross_bindir}/objcopy
+#%%{_cross_prefix}%%{_cross_bindir}/strings
+#%%{_cross_prefix}%%{_cross_bindir}/addr2line
+%{_cross_prefix}%{_cross_bindir}/nm
+#%%{_cross_prefix}%%{_cross_bindir}/size
+%{_cross_prefix}%{_cross_bindir}/ld
+#%%{_cross_prefix}%%{_cross_bindir}/elfedit
+%{_cross_prefix}%{_cross_bindir}/ranlib
+%{_cross_prefix}%{_cross_bindir}/readelf
+%{_cross_prefix}%{_cross_bindir}/strip
+
+# These duplicates are not found in the normal binutils
+#%%{_cross_prefix}%%{_bindir}/%%{_tuple_name}dwp
+%{_cross_prefix}%{_bindir}/%{_tuple_name}gprof
+%{_cross_prefix}%{_bindir}/%{_tuple_name}ld.bfd
+#%%{_cross_prefix}%%{_bindir}/%%{_tuple_name}ld.gold
+%{_cross_prefix}%{_bindir}/%{_tuple_name}c++filt
+%{_cross_prefix}%{_bindir}/%{_tuple_name}objdump
+%{_cross_prefix}%{_bindir}/%{_tuple_name}as
+%{_cross_prefix}%{_bindir}/%{_tuple_name}ar
+%{_cross_prefix}%{_bindir}/%{_tuple_name}objcopy
+%{_cross_prefix}%{_bindir}/%{_tuple_name}strings
+%{_cross_prefix}%{_bindir}/%{_tuple_name}addr2line
+%{_cross_prefix}%{_bindir}/%{_tuple_name}nm
+%{_cross_prefix}%{_bindir}/%{_tuple_name}size
+%{_cross_prefix}%{_bindir}/%{_tuple_name}ld
+%{_cross_prefix}%{_bindir}/%{_tuple_name}elfedit
+%{_cross_prefix}%{_bindir}/%{_tuple_name}ranlib
+%{_cross_prefix}%{_bindir}/%{_tuple_name}readelf
+%{_cross_prefix}%{_bindir}/%{_tuple_name}strip
+
+%{_cross_prefix}%{_cross_libdir}/ldscripts/*
+#%%{_cross_prefix}%{_cross_libdir}/bfd-plugins/libdep.so
+"/opt/cross/aarch64-mariner-linux-gnu/lib/bfd-plugins/libdep.so"
+
+%{_cross_prefix}%{_mandir}/man1/%{_tuple_name}readelf.1
+%{_cross_prefix}%{_mandir}/man1/%{_tuple_name}windmc.1
+%{_cross_prefix}%{_mandir}/man1/%{_tuple_name}ranlib.1
+%{_cross_prefix}%{_mandir}/man1/%{_tuple_name}gprof.1
+%{_cross_prefix}%{_mandir}/man1/%{_tuple_name}strip.1
+%{_cross_prefix}%{_mandir}/man1/%{_tuple_name}c++filt.1
+%{_cross_prefix}%{_mandir}/man1/%{_tuple_name}as.1
+%{_cross_prefix}%{_mandir}/man1/%{_tuple_name}objcopy.1
+%{_cross_prefix}%{_mandir}/man1/%{_tuple_name}elfedit.1
+%{_cross_prefix}%{_mandir}/man1/%{_tuple_name}strings.1
+%{_cross_prefix}%{_mandir}/man1/%{_tuple_name}nm.1
+%{_cross_prefix}%{_mandir}/man1/%{_tuple_name}ar.1
+%{_cross_prefix}%{_mandir}/man1/%{_tuple_name}ld.1
+%{_cross_prefix}%{_mandir}/man1/%{_tuple_name}dlltool.1
+%{_cross_prefix}%{_mandir}/man1/%{_tuple_name}addr2line.1
+%{_cross_prefix}%{_mandir}/man1/%{_tuple_name}windres.1
+%{_cross_prefix}%{_mandir}/man1/%{_tuple_name}size.1
+%{_cross_prefix}%{_mandir}/man1/%{_tuple_name}objdump.1
+
+# These are the -devel files, we are not generating them.
```

Comment:
There is a lot of commented out content here.  Essentially dead code.  Do we need the headers?